flexget/components/managed_lists/lists/pending_list/pending_list.py

```diff
@@ -19,15 +19,24 @@ class PendingListSet(MutableSet):
     def _db_list(self, session):
         return (
             session.query(db.PendingListList)
-            .filter(db.PendingListList.name == self.config)
+            .filter(db.PendingListList.name == self.list_name)
             .first()
         )
 
     def __init__(self, config):
-        self.config = config
+        if not isinstance(config, dict):
+            config = {'list_name': config}
+        self.list_name = config.get('list_name')
+
+        self.filter_approved = True
```

Comment:
I meant an actual `Enum`, something like this:
```python
from enum import Enum

class PendingListInclude(Enum):
    pending = False
	approved = True
	all = None
```
Then use it like this:
```python
self.include = PendingListFilter(config['include'])
...
query = self._db_list(session).entries.order_by(db.PendingListEntry.added.desc())
if self.include is not PendingListInclude.all:
    query = query.filter(db.PendingListEntry.approved == self.include.value)
```

Reply:
@liiight I could do that but  it is  only used in two places.. feels nore complicated then it needs to be?

Reply:
Feels nicer to me, but ok. Python enums are great